evap/staff/templates/staff_course_comments_quick.html

```diff
@@ -0,0 +1,436 @@
+                </button>
+                <button role="button" data-action="make_private" class="btn btn-sm btn-outline-secondary" disabled="">
+                    {% trans 'private' %}
+                </button>
+                <button role="button" data-action="hide" class="btn btn-sm btn-outline-secondary">
+                    {% trans 'no' %}
+                </button>
+            </div>
+            <div data-action-set="repeat" class="d-none">
+            {% if next_course %}
+                <a href="{% url 'staff:course_comments' next_course.semester.id next_course.id %}" data-action="next-course" class="btn btn-sm btn-primary">
+                    {% trans 'Review next course' %}
+                </a>
+            {% endif %}
+                <button role="button" data-action="repeat-all" class="btn btn-sm btn-outline-primary">
+                    {% trans 'Show all again' %}
+                </button>
+                <button role="button" data-action="repeat-unreviewed" class="btn btn-sm btn-outline-primary">
+                    {% trans 'Show unreviewed' %}
+                </button>
+            </div>
+        </div>
+    </div>
+{% endblock %}
+
+{% block additional_javascript %}
+    <script type="text/javascript">
+        (function($) {
+            function Layer() {
+                this.setup.apply(this, arguments);
+            }
+
+            Layer.prototype = {
+                setup: function(items, reference) {
+                    this.items = items;
+                    this.reference = reference;
+                    this.active = items.filter(".active");
+
+                    items.on("transitionend", function() {
+                        $(this).removeClass("to-left to-right");
+                    });
+                },
+
+                slide: function(element, direction) {
+                    if(this.active.is(element)) {
+                        return;
+                    }
+
+                    if(this.active.length) {
+                        let reversed = direction === "left"? "right" : "left";
+                        this.active.css("top", this.active.position().top);
+                        this.active.removeClass("active to-left to-right");
+                        this.active.addClass("to-" + reversed);
+                    }
+
+                    if(this.reference) {
+                        let reference = this.reference.items.filter("[data-id=" + element.data("reference") + "]");
+                        this.reference.slide(reference, direction);
+                    }
+
+                    this.active = element;
+
+                    if(this.active.length) {
+                        this.active.removeClass("to-left to-right");
+                        this.active.addClass("to-" + direction);
+                        this.active.position();
+                        this.active.addClass("active");
+                    }
+                }
+            };
```

Comment:
is it possible to remove this whole class and put code into the other classes `slide` method that makes the three elements slide if necessary? would that be more straightforward?